batchflow/models/torch/base.py

```diff
@@ -1158,7 +1208,7 @@ def _add_output_op(self, inputs, oper, name, attr_prefix, **kwargs):
             output = inputs.argmax(dim=class_axis)
```

Comment:
the line #1185 
`raise TypeError("Network output is expected to be a Tensor, but given {}".format(type(inputs)))`
should rather be 
`raise TypeError("Network output is expected to be a Tensor, but given {}".format(type(tensor)))`
because `inputs` is always a list or a tuple